src/components/patterns/headingList.tsx

```diff
@@ -70,7 +70,8 @@ const HeadingList = () => {
     }
 
     const observer = new IntersectionObserver(observerCallback, {
-      threshold: 1.0,
+      rootMargin: "0px 0px -80% 0px",
```

Comment:
FYI: Revert this to the previous config since we only want the active state to change to the following heading when the heading is in the upper 20% of the window.